controllers/metallb_controller.go

```diff
@@ -81,10 +82,14 @@ func (r *MetallbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		// Error reading the object - requeue the request.
 		return ctrl.Result{}, err
 	}
+	status.UpdateStatus(context.TODO(), r.Client, instance, status.ConditionProgressing, "", "")
 
 	err = r.syncMetalLBResources(instance)
 	if err != nil {
+		status.UpdateStatus(context.TODO(), r.Client, instance, status.ConditionDegraded, "FailedToSyncMetalLBResources", err.Error())
 		return ctrl.Result{}, err
+	} else {
+		status.UpdateStatus(context.TODO(), r.Client, instance, status.ConditionAvailable, "", "")
```

Comment:
I believe in order to declare condition available we must ensure that all the operands are running.
Here we only know that we deployed the resources necessary to run them.

Reply:
done